lib/services/networkManagement2/lib/models/index.d.ts

```diff
@@ -4692,15 +5177,13 @@ export interface Operation {
  * resource group. This name can be used to access the resource.
  * @member {string} [etag] A unique read-only string that changes whenever the
  * resource is updated.
- * @member {object} [tags] Resource tags.
  */
 export interface PatchRouteFilterRule extends SubResource {
   access: string;
   communities: string[];
   readonly provisioningState?: string;
   readonly name?: string;
   readonly etag?: string;
-  tags?: { [propertyName: string]: string };
```

Comment:
Removing an interface property is a breaking change. Requires major version increment.